### PR TITLE
Added iOS section to libraries.html

### PR DESCRIPTION
**Pull request by ToddKerpelman**
Added descriptions for 3 libraries; the Maps SDK for iOS, the OpenInGoogleMapsController, and the iOS Utils project,.

### CODE REVIEW

File: libraries.html [hunk 1]
```diff
@@ -84,6 +84,36 @@ <h1 class="logo">Google Maps API Libraries</h1>
         </tbody>
       </table>
 
+      <h2>Google Maps iOS API</h2>
```
Comment (broady) [inline libraries.html]: Hm, the product name is "Google Maps SDK for iOS".

The URL scheme docs are under that doc set:
https://developers.google.com/maps/documentation/ios/urlscheme

File: libraries.html [hunk 2]
```diff
@@ -84,6 +84,36 @@ <h1 class="logo">Google Maps API Libraries</h1>
...
+            <th class="title">Library</th>
+            <th>Summary</th>
+            <th class="title">Maintainer</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td class="title"><a href="https://developers.google.com/maps/documentation/ios/">The Google Maps SDK for iOS</a></td>
+            <td>A multipurpose library that allows you to embed Google Maps, Stret View, Indoor Maps, and much more inside your native iOS application.</td>
+            <td class="title">The Google Maps team</a></td>
```
Comment (broady) [inline libraries.html]: Cool. I actually like having this entry here.

@brendankenny @markmcd @domesticmouse FYI

Comment (domesticmouse) [inline libraries.html]: lgtm

File: libraries.html (hunk 1 above)
Comment (ToddKerpelman) [inline libraries.html]: Happy to change it.. this just seemed more consistent with all the other headers. Just let me know.

Comment (domesticmouse) [inline libraries.html]: It was named like that on advice from legal. Please change.

Comment (broady) [inline libraries.html]: Yep, please change to the product name. The others are our product names - naming is not consistent for various reasons :)

**Pull request merged**